client/wfnews-war/src/main/angular/src/app/components/panel-wildfire-stage-of-control/panel-wildfire-stage-of-control.component.ts

```diff
@@ -355,4 +378,9 @@ export class PanelWildfireStageOfControlComponent extends CollectionComponent im
     removeFromWatchlist (incident: any) {
       this.watchlistService.removeFromWatchlist(incident.incidentNumberLabel)
     }
+
+    onClickBookmark(event:Event) {
+      console.log(event)
```

Comment:
Remember to clear out the console.logs

Reply:
Also, is the event needed for anything else or just to stop the onclick event?

Reply:
nope, It is for stopping the onClick of openPreview(). Making clicking bookmark not expanding preview panel.